external-crates/move/move-compiler/tests/move_check/typing/constraints_not_satisfied_type_arguments_internal_pack.exp

```diff
@@ -40,3 +40,11 @@ error[E05001]: ability constraint not satisfied
   │         Cannot ignore values without the 'drop' ability. The value must be used
   │         The type '0x8675309::M::Box<0x8675309::M::R>' does not have the ability 'drop'
 
+warning[W09005]: dead or unreachable code
+  ┌─ tests/move_check/typing/constraints_not_satisfied_type_arguments_internal_pack.move:9:9
+  │
+9 │         Box<R>{ f: R{} };
+  │         ^^^^^^^^^^^^^^^^ Unreachable code. This statement (and any following statements) will not be executed.
```

Comment:
We should not emit this error as it is a duplicate dead code error 
(with the `Invalid use of a divergent expression.` above)